hoomd/mpcd/ConfinedStreamingMethod.h

```diff
@@ -101,14 +102,20 @@ void ConfinedStreamingMethod<Geometry>::stream(unsigned int timestep)
         m_validate_geom = false;
         }
 
+    if (m_mpcd_pdata->getN() > m_bounced.getNumElements())
+        {
+        GPUArray<unsigned char> bounced(m_mpcd_pdata->getN());
+        m_bounced.swap(bounced);
+        }
+    
     if (m_prof) m_prof->push("MPCD stream");
 
     const BoxDim& box = m_mpcd_sys->getCellList()->getCoverageBox();
 
     ArrayHandle<Scalar4> h_pos(m_mpcd_pdata->getPositions(), access_location::host, access_mode::readwrite);
     ArrayHandle<Scalar4> h_vel(m_mpcd_pdata->getVelocities(), access_location::host, access_mode::readwrite);
     const Scalar mass = m_mpcd_pdata->getMass();
-
+    ArrayHandle<unsigned char> h_bounced(m_bounced, access_location::host, access_mode::overwrite);
```

Comment:
Try to group things logically together with whitespace for readability. This should be grouped with lines 115-117, since it is a particle quantity, with a blank line after it.